seacrowd/sea_datasets/massive/massive.py

```diff
@@ -0,0 +1,508 @@
+    'social_post', 
+    'recommendation_events', 
+    'transport_taxi', 
+    'takeaway_order', 
+    'music_query', 
+    'calendar_query', 
+    'lists_query', 
+    'qa_currency', 
+    'recommendation_movies', 
+    'general_joke', 
+    'recommendation_locations', 
+    'email_querycontact', 
+    'lists_remove', 
+    'play_audiobook', 
+    'email_addcontact', 
+    'lists_createoradd', 
+    'play_radio', 
+    'qa_stock', 
+    'alarm_query', 
+    'email_sendemail', 
+    'general_quirky', 
+    'music_likeness', 
+    'cooking_recipe', 
+    'email_query', 
+    'datetime_query', 
+    'transport_traffic', 
+    'play_podcasts', 
+    'iot_hue_lightchange', 
+    'calendar_remove', 
+    'transport_query', 
+    'transport_ticket', 
+    'qa_factoid', 
+    'iot_cleaning', 
+    'alarm_set', 
+    'datetime_convert', 
+    'iot_hue_lightoff', 
+    'qa_definition', 
+    'music_dislikeness'
+]
+
+_TAGS = ['O', 'B-food_type', 'B-movie_type', 'B-person', 'B-change_amount', 'I-relation', 'I-game_name', 'B-date', 'B-movie_name', 
+    'I-person', 'I-place_name', 'I-podcast_descriptor', 'I-audiobook_name', 'B-email_folder', 'B-coffee_type', 'B-app_name', 'I-time', 
+    'I-coffee_type', 'B-transport_agency', 'B-podcast_descriptor', 'I-playlist_name', 'B-media_type', 'B-song_name', 'I-music_descriptor', 
+    'I-song_name', 'B-event_name', 'I-timeofday', 'B-alarm_type', 'B-cooking_type', 'I-business_name', 'I-color_type', 'B-podcast_name', 
+    'I-personal_info', 'B-weather_descriptor', 'I-list_name', 'B-transport_descriptor', 'I-game_type', 'I-date', 'B-place_name', 'B-color_type', 
+    'B-game_name', 'I-artist_name', 'I-drink_type', 'B-business_name', 'B-timeofday', 'B-sport_type', 'I-player_setting', 'I-transport_agency', 
+    'B-game_type', 'B-player_setting', 'I-music_album', 'I-event_name', 'I-general_frequency', 'I-podcast_name', 'I-cooking_type', 'I-radio_name', 
+    'I-joke_type', 'I-meal_type', 'I-transport_type', 'B-joke_type', 'B-time', 'B-order_type', 'B-business_type', 'B-general_frequency', 
+    'I-food_type', 'I-time_zone', 'B-currency_name', 'B-time_zone', 'B-ingredient', 'B-house_place', 'B-audiobook_name', 'I-ingredient', 
+    'I-media_type', 'I-news_topic', 'B-music_genre', 'I-definition_word', 'B-list_name', 'B-playlist_name', 'B-email_address', 'I-currency_name', 
+    'I-movie_name', 'I-device_type', 'I-weather_descriptor', 'B-audiobook_author', 'I-audiobook_author', 'I-app_name', 'I-order_type', 
+    'I-transport_name', 'B-radio_name', 'I-business_type', 'B-definition_word', 'B-artist_name', 'I-movie_type', 'B-transport_name', 'I-email_folder', 
+    'B-music_album', 'I-house_place', 'I-music_genre', 'B-drink_type', 'I-alarm_type', 'B-music_descriptor', 'B-news_topic', 'B-meal_type', 
+    'I-transport_descriptor', 'I-email_address', 'I-change_amount', 'B-device_type', 'B-transport_type', 'B-relation', 'I-sport_type', 'B-personal_info'
+]
+
+
+class MASSIVE(datasets.GeneratorBasedBuilder):
```

Comment:
can you modify the class name to MASSIVEDataset for consistency to other datasets?